tests/query_planner/test_query_planner.py

```diff
@@ -324,3 +324,39 @@ async def testable_generate_plans():
         testable_generate_plans,
         "generate plan for a typical query graph (50k KPs)",
     )
+
+
+@pytest.mark.asyncio
```

Comment:
I am trying to keep the tests marked with @pytest.longrun at the bottom of this file. Can you move this test above those?